### ISSUE TITLE
Where are the Ceylon communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/ceylon#68]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Ceylon supported by one or more large organizations?
* Does Ceylon have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Ceylon? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Ceylon? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Ceylon? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Ceylon?
* Are there popular newsletters for Ceylon?
* Is Ceylon taught at programming bootcamps? (If so, what are some examples?)
* Is Ceylon taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Ceylon?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by petertseng** [comment]
Here are some answers that I know. If I omit a question, you may assume my answer is "I don't know". If there is a particular question that you wanted to confirm my answer is "I don't know", you can ask about it and I'm happy to confirm whether I know.

> Is Ceylon supported by one or more large organizations?

Formerly Red Hat, and currently the Eclipse Foundation as of https://ceylon-lang.org/blog/2017/08/21/eclipse-ceylon/ . I don't know and cannot speculate on how much support either of these organisations provides.

> Does Ceylon have an official community manager?

Sorry, I couldn't determine that anyone listed in https://ceylon-lang.org/community/team/ officially has this designation, and I don't know the community well enough to give an answer from sources other than this page

> Do you know of specific communities (online or offline) that are enthusiastic about Ceylon? 

https://gitter.im/ceylon/user I guess. The project's own recommendations are https://ceylon-lang.org/community/

> Are there popular conferences for Ceylon? 

I was very excited to list all the events from https://ceylon-lang.org/community/events/ here, but there are no events listed there. The most recent https://ceylon-lang.org/community/conferences/ceylon-tour-paris-2014/ was in 2014 apparently.

**Comment by kytrinyx** [comment]
Thank you @petertseng!
